almanac.lic

```diff
@@ -12,6 +12,8 @@ class Almanac
     UserVars.almanac_last_use ||= Time.now - 600
     @no_use_scripts = settings.almanac_no_use_scripts
     @almanac_skills = settings.almanac_skills
+    @almanac = settings.almanac_noun
+    @almanac_container = settings.almanac_container
```

Comment:
❓ Where is this used?

Reply:
Ahhhh this this making me crazy.   Finished the rest of the changes.

Reply:
aborted the almanac_container.  Forgot to remove it here.